lib/resolvers/PackageResolver.ts

```diff
@@ -105,6 +117,24 @@ export class PackageResolver {
     });
   }
 
+  /**
+   * Retrieves the latest Angular Material version remotely.
+   */
+  private static _retrieveLatestVersion(): Promise<string> {
+    return new Promise((resolve, reject) => {
+      request('http://material.angularjs.org/docs.json', (error, response, body) => {
```

Comment:
Yeah we use it for testing, because we need to gather all versions. In your case we only need the latest version and we don't want to rely on the documentation firebase (which can change) in production.

You could just do it the "normal" way
```ts
// Can print stderr to console (unwanted)
let latestVersion = execSync('npm view angular-material version').toString();
// Doesn't print anything out. But may not work with `NPM` on Windows
let latestVersion = spawnSync('npm', ['view', 'angular-material', 'version']).stdout.toString();
```